### DISCUSSION TITLE
Add additional VehicleViewDirections to be able to download more pictures

### DISCUSSION THREAD

**Discussion by zimmermannb** [bimmerconnected/bimmer_connected#660] [Feature request] [answered]
Hi, 

i did some research and found a bunch of more valid strings that can be used to download images using the API. 
I also discovered, that the API accepts the parameter "small":[True|False] to allow the download of smaller versions of the images. 

Valid Directions are: 

AngleSideViewForty
AngleSideViewSixty
Dashboard
Driverdoor
FrontLeft
FrontRight
FrontView
RearLeft
RearRight
RearView
SideViewLeft
VehicleStatus

Would be great so see those values make it into bimmer_connected. Would be even better if the HomeAssistant-Integration would be able to show the images.

**Comment by rikroe** [comment, marked as answer]
I've added the directions with #666 / 0.16.4. Maybe I'll find some time to implement image entities for HA.
